scw/config.go

```diff
@@ -143,6 +143,10 @@ func (c *Config) String() string {
 	return string(configRaw)
 }
 
+func (c *Config) Empty() bool {
```

Comment:
`IsEmpty` Vs `Empty` @QuentinBrosse a third opinion ?

Reply:
I vote for `IsEmpty`.

"If config is empty, do something" could be translated to:
```go
if config.IsEmpty() {
    // do something
}
```